src/test/java/dev/openfeature/sdk/integration/StepDefinitions.java

```diff
@@ -12,24 +15,35 @@
 import dev.openfeature.sdk.Structure;
 import dev.openfeature.sdk.Value;
 import io.cucumber.java.BeforeAll;
+import io.cucumber.java.Before;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+import io.cucumber.java.en.And;
+
+import java.io.File;
+import java.io.FileWriter;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
 
 public class StepDefinitions {
```

Comment:
Let's annotate this with `@Slf4j` get logger enabled

Reply:
thanks, done